keyutil/jwk.go

```diff
@@ -74,55 +91,60 @@ func FromPrivateKey(fields JWK, pkey crypto.PrivateKey) (JWK, error) {
 
 // JWK provides basic data and usage for a JWK.
 type JWK struct {
-	KeyType   string // 'kty'
+	KeyType   string // 'kty' - "EC", "RSA", "oct"
 	Algorithm string // 'alg'
 	KeyID     string // 'kid'
-	raw       json.RawMessage
+	jtype     any    // the type of JWK based on KeyType.
```

Comment:
nit: the field `jtype` holds the concrete value. The name sounds like it's some type only, from direct look, but I think it's okay since it's not exposed 🤔

Reply:
sgtm, thanks!